tasks/utils/i18n_verify_keys.js

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs';
+import path from 'path';
+import Promise from 'bluebird';
+import globProm from 'glob-promise';
```

Comment:
Good point. After our disucssion on IRC, we agreed that grunt is needed either. So, updated it to just 'glob' + 'promisfy' as we agreed.